First pull request

Neven-Benko opened the pull request.

Commit:
initial commit

Commit:
- removed extra divs
- fixed footer height

Commit:
additional fixes of ehader (height, input height, icon color)

Commit:
2nd commit: reduced number of html elements, changed class names for consistency, adjusted heights and colors

Commit:
-added CSS custom variables for colors
-removed dead CSS properties and fixed widths of elements
-form input types changed to emails and password
-grid instead of flex for main layout
-removed redundant HTML elements
-still unable to make form and header fully responsive

tonkec commented on auth_Styles.css:
```diff
@@ -0,0 +1,120 @@
+}
+
+.form-flex {
+    width: auto;
```
hm jel ovo defaultno tako?

tonkec commented on auth_Styles.css:
```diff
@@ -0,0 +1,120 @@
+}
+
+footer {
+    width: 100%;
```
ne treba ti 100% ovdje, mislim

tonkec commented on auth_Styles.css:
```diff
@@ -0,0 +1,120 @@
+.button-auth {
+    width: 100%;
+    max-width: 400px;
+    height: 2.5em;
```
izbjegavaj fixne vrijednosti, sto ako se tekst u buttonu promijeni?

tonkec commented on header_styles.css:
```diff
@@ -0,0 +1,99 @@
+
+/* LAYOUT ============================================== */
+
+.main-wrapper {
```
zasto se ovdje stilovi ponavljaju kao u auth_Styles.css fajlu?

tonkec commented on header_styles.css:
```diff
@@ -0,0 +1,99 @@
+    padding: 0.9em;
+    height: 2.5rem;
+    width: 100%;
+    max-width: 475px;
```
zasto taj magicni broj?

Commit:
Auth styles:
Removed footer width property
Removed input & button max width properties
Dont understand this comment:
--------------------------------------------------------------------------------
In auth_Styles.css:
> +    grid-template-rows: auto 1fr auto;
+    grid-template-columns: 100%;
+}
+
+main {
+    margin: auto;
+}
+
+.nav-flex {
+    display: flex;
+    align-items: center;
+    justify-content: space-between;
+}
+
+.form-flex {
+    width: auto;
hm jel ovo defaultno tako?
------------------------------------------------------------------------------------------------------

Header Styles:
Removed redundant styling
Removed magical numbers

Commit:
Removed fixed heights on buttons and inputs

Neven-Benko committed:
Merge branch 'kodiraonica:main' into pages/auth
